AmplicaLabs/account-service
# Implement k6 load test scaffolding

# Goal 
The goal of this PR is to implement the initial scaffolding for the k6 load testing.

# Changes

- [x] add `k6-scripts` directory
- [x] add `health-check.js` as simple demonstration for endpoint testing.
- [x] add `account-service-load.js` as example of auto-generated OpenAPI test script for further editing.

# Testing

1. Install `k6`, see `README.md` for link to installation instructions.
2. Run the `health-check.js` and see that it executes correctly.
